packages/edge/README.md

```diff
@@ -0,0 +1,55 @@
+
+# React Native AEP Edge Network
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedge.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedge) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedge)](https://www.npmjs.com/package/@adobe/react-native-aepedge)
+
+## Usage
+
+### [Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension)
+
+#### Importing the extension:
+```javascript
+import {AEPEdge} from '@adobe/react-native-aepedge';
+```
+
+##### Getting the extension version:
+
+```javascript
+AEPEdge.extensionVersion().then(version => console.log("AdobeExperienceSDK: AEPEdge version: " + version));
+```
+##### Create Experience Event from Dictionary:
+
+```javascript
+var xdmData  = {"eventType" : "SampleXDMEvent"};
+let experienceEvent = ExperienceEvent(xdmData);
```

Comment:
should this be `let experienceEvent = new AEPExperienceEvent(xdmData);` ? same for all examples below